lib/rucio/core/rule.py

```diff
@@ -1849,7 +1845,7 @@ def get_injected_rules(total_workers, worker_number, limit=100, blocked_rules=[]
 
 
 @read_session
-def get_stuck_rules(total_workers, worker_number, delta=600, limit=10, blocked_rules=[], *, session: "Session"):
+def get_stuck_rules(total_workers: int, worker_number: int, delta: int = 600, limit: int = 10, blocked_rules: Sequence[str] = [], *, session: "Session") -> list[tuple[str, str]]:
```

Comment:
Exactly the same comment as above for the return type. Why do we do such things? :smiling_face_with_tear: